frontend/packages/akr/src/pages/clerk/ClerkSendEmailPage.tsx

```diff
@@ -102,6 +103,7 @@ const ControlButtons = ({
   return (
     <div className="columns gapped flex-end">
       <CustomButton
+        disabled={isLoading}
```

Comment:
Mietin että olisko tä looginen olla myös omassa muuttujassa `cancelDisabled` täällä vastaavasti kuin `submitDisabled`. Mutta voi olla näinkin.

Reply:
Mjoo, mergaan näin mutta tästä voidaan jossaki vaiheessa vaikka jutella yleisesti. Oon ite tykänny et käytän vähemmän noita välimuutujia jos voi vaihtoehtoisesta vaan passaa jo olemassaolevan muuttujan tai about max. 2 muuttujaa niin ei tarvi lähteä trackkaan sitä arvoa koodista